### PR TITLE
blog: v0.17.1.5 cli release notes

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: _posts/2020-11-25-monero-0.17.1.5-released.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Monero 0.17.1.5 'Oxygen Orion' Point Release"
+summary: Point release containing Dandelion++ perfomance
```
Comment (erciccione) [inline _posts/2020-11-25-monero-0.17.1.5-released.md:4]: Maybe you meant "performance improvements"?

Comment (selsta) [reply _posts/2020-11-25-monero-0.17.1.5-released.md]: yep, thanks

**Review by erciccione — APPROVED** [review]
Hashes to be verified.

**Comment by binaryFate** [comment]
Hashes ok

**Pull request merged**